test/utils.test.js

```diff
@@ -64,6 +64,44 @@ describe('utils', () => {
         });
     });
 
+    describe('#isValidAccountID', () => {
+        test('returns true if string is valid', () => {
+            const value = 'abcd-1234.fffff';
+            const isValidAccountID = utils.isValidAccountID(value);
+            expect(isValidAccountID).toBe(true);
+        });
+
+        test('returns false if string has any capital letters', () => {
```

Comment:
Are uppercase letters invalid? I missed this part. Should we coerce upper to lower?

Reply:
Coerce to lower or pass specific error message. This happened to me when I was using a library and it took forever to figure out why my address was incorrect. For some reason wherever I was getting my address from had letters in lower/uppercase and the address verifier wanted the opposite.